Lib/fontbakery/profiles/gdef.py

```diff
@@ -0,0 +1,116 @@
+from fontbakery.callable import check
+from fontbakery.checkrunner import FAIL, PASS, WARN
+from fontbakery.message import Message
+from fontbakery.utils import pretty_print_list
+
+# used to inform get_module_profile whether and how to create a profile
+from fontbakery.fonts_profile import profile_factory # NOQA pylint: disable=unused-import
+
+from fontTools import unicodedata
+
+@ check(
+  id = 'com.google.fonts/check/gdef_spacing_marks',
+  rationale ="""
+  Glyphs in the GDEF mark glyph class should be non-spacing.
+  Spacing glyphs in the GDEF mark glyph class may have incorrect
+  anchor positioning that was only intended for building composite glyphs
+  during design.
+  """
```

Comment:
Use 4-space indentation on the contents of the rationale string, please. Otherwise, the rationale text gets clipped by the text terminal reporter.